src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working ;( ");

```

Comment:
Similarly, the message in the 'stopWork' method should clearly indicate the type of machine that has stopped working. Consider rephrasing the message to: "Bulldozer has stopped working."